hpx/runtime/threads/coroutines/detail/context_posix.hpp

```diff
@@ -396,7 +408,6 @@ namespace hpx { namespace threads { namespace coroutines
             // declare m_stack_size first so we can use it to initialize m_stack
             std::ptrdiff_t m_stack_size;
             void * m_stack;
-            void * cb_;
```

Comment:
why has this been removed?

Reply:
@sithhell do we still need it?

Reply:
oh right, missed that we don't need it anymore...